.github/workflows/release.yml

```diff
@@ -14,3 +14,25 @@ jobs:
       - run: yarn publish
         env:
           NPM_AUTH_TOKEN: ${{ secrets.YARN_NPM_AUTH_TOKEN }}
+
+  build-binary:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.24.1'
+      - name: Install project dependencies
+        run: yarn install --immutable
+      - name: Bundle library
+        run: yarn build
+      - name: Create standalone binary
+        run: yarn dist-standalone
```

Comment:
I think this command will fail as it will try to use node 10.24.1 by default, which `nexe` has no release for.

Also this job shares multiple steps with the `standalone-binary-test` job, would it be possible to factorize them? (we could have "build", "test" and "release" jobs all relying on ci artifacts to pass the built bundle between each other if github action supports it)

Reply:
that's a good catch, I missed adding the target in here. 

As for the sharing of artifacts, I'd have to check how that works. It's not a slow step and it's not blocking the release so I think it's fine. We can look into speeding this up later, I think